### DISCUSSION TITLE
SMD Heat Plate recommendition

### DISCUSSION THREAD

**Discussion by bjoerns1983** [OMOTE-Community/OMOTE-Firmware#45] [General]
Hi there,
together with some friends i order 10 PCB's and all the Stuff needed to produce 10 Omotes.
Till now i have no big experience doing SMD soldering, so i would to buy a heatplate.

Now i would like your opinion. On one Hand there are these small SMD Heat Plates like the MHP30 from the Video, but you could also get something bigger like the Uyue 946c.

What would you recommend? Where are the pros and cons?

**Comment by CoretechR** [comment]
Hi, this was the first project that I tried reflow soldering and I chose the mini hotplate because I don't have a lot of space. Using a bigger hot plate should make the process easier as you don't have to move the PCB over the plate. I can't really recommend you a specific hot plate but there are lots of cheap options to choose from.

**Comment by bjoerns1983** [comment]
But how did you solder he buttons? By Hand or a Hot Air Gun?

**Reply by CoretechR** [reply]
By hand, but hot air should work fine as well.
